pkg/policy/controller.go

```diff
@@ -319,15 +347,36 @@ func (pc *PolicyController) Run(workers int, stopCh <-chan struct{}) {
 	logger.Info("starting")
 	defer logger.Info("shutting down")
 
-	if !cache.WaitForCacheSync(stopCh, pc.pListerSynced, pc.cpvListerSynced, pc.nspvListerSynced, pc.nsListerSynced,pc.grListerSynced) {
-		logger.Info("failed to sync informer cache")
-		return
+	if os.Getenv("POLICY-TYPE") == common.PolicyReport {
+		if !cache.WaitForCacheSync(stopCh, pc.pListerSynced, pc.nsListerSynced) {
+			logger.Info("failed to sync informer cache")
+			return
+		}
+	} else {
+		if !cache.WaitForCacheSync(stopCh, pc.pListerSynced, pc.cpvListerSynced, pc.nspvListerSynced, pc.nsListerSynced) {
+			logger.Info("failed to sync informer cache")
+			return
+		}
+
+	}
+
+	if os.Getenv("POLICY-TYPE") == common.PolicyReport {
+		go func(pc *PolicyController) {
+			for k := range time.Tick(constant.PolicyReportPolicyChangeResync) {
+				pc.log.V(2).Info("Policy Background sync at", "time", k.String())
```

Comment:
According to GoDoc, `time.Tick()` is a convenience wrapper for `NewTicker`, and 
> be aware that without a way to shut it down the underlying, Ticker cannot be recovered by the garbage collector; it "leaks".

Let's create one ticker out of for loop and use `switch` to invoke the logic,
```
ticker := time.Tick(10 * time.Second)
for {
  select {
  case <-ticker:
  // your logic here
  }
}
```